property.sql

```diff
@@ -0,0 +1,37 @@
+CREATE TABLE IF NOT EXISTS `properties` (
+	`id` INT NOT NULL AUTO_INCREMENT,
+	`name` VARCHAR(255) NOT NULL,
+	`interior` VARCHAR(255) NOT NULL,
+	`furnished` BOOLEAN NOT NULL,
+	`owners` JSON,
+	`garage` BOOLEAN NOT NULL,
+	`coords` JSON NOT NULL,
+	`price` INT NOT NULL,
+	`rent` INT NOT NULL,
+	`rent_date` TIMESTAMP DEFAULT NULL,

```

Comment:
Is this the date the rent is due, or when renting started?

Reply:
Consider renaming to rent_expiration

Reply:
makes more sense thanks lol